ajax/libs/dompurify/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "dompurify",
+  "description": "DOMPurify is a DOM-only, super-fast, uber-tolerant XSS sanitizer for HTML, MathML and SVG. It's written in JavaScript and works in all modern browsers (Safari, Opera (15+), Internet Explorer (10+), Firefox and Chrome - as well as almost anything else using Blink or WebKit). DOMPurify is written by security people who have vast background in web attacks and XSS. Fear not.",
+  "version": "0.7.2",
+  "filename": "purify.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/cure53/DOMPurify.git"
+  },
+  "keywords": [
+    "dom",
+    "xss",
+    "html",
+    "svg",
+    "mathml",
+    "security",
+    "secure",
+    "sanitizer",
+    "sanitize",
+    "filter",
+    "purify"
+  ],
+  "author": "Mario Heiderich <mario@cure53.de> (https://cure53.de/)",
+  "license": [
```

Comment:
@LeaYeh yes, we added lib with two license before.
@PeterDaveHello do we must modify `license` to `licenses`?